src/App.js

```diff
@@ -1,4 +1,4 @@
-import 'antd/dist/antd.less';
+import 'antd/dist/antd.min.css'

```

Comment:
should we put antd.css in app component?

Reply:
yep!, we should because if we put it to `index.js it will overwrite all style of `general.scss`